workflow/exp/exp.ens_conus12km

```diff
@@ -48,7 +47,10 @@ export LBC_EXTRN_MDL_BASEDIR="${COMINgefs}/gefs.@Y@m@d/@H"
 export LBC_EXTRN_MDL_NAME_PATTERN="pgrb2ap5/gep#gmem#.t@Hz.pgrb2a.0p50.ffHHH"
 export LBC_EXTRN_MDL_NAME_PATTERN_b="pgrb2bp5/gep#gmem#.t@Hz.pgrb2b.0p50.ffHHH"
 #
-export RETRO_PERIOD='2024052623-2024052623'
+export RUN_PERIOD="2024052623-2024052623"
+export CYCLEDEF_IC="   &STARTYEAR;&STARTMONTH;&STARTDAY;2300 &ENDYEAR;&ENDMONTH;&ENDDAY;2300 06:00:00 "
```

Comment:
Just think of this, do we really need individual year, month, and day?

Can we make it simple using something as follows?
```
export CYCLEDEF_IC="   &START_PDY;2300 &END_PDY;2300 06:00:00 "
```

Reply:
We have another way to set up cycledef which needs to know year, month, day separately.